README.md

```diff
@@ -39,8 +39,25 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
+      - name: Get tokens
+        id: gettokens
+        run: |
+          echo '${{ toJson(secrets) }}' > secrets.json
+          echo ::set-output name=tokens::$(cat secrets.json)
+      - name: Run action
+        run: |
+          NPM_TOKEN=$(echo '${{ steps.gettokens.outputs.tokens }}' | jq -R '. | fromjson | to_entries[] | select((.key | startswith("NPM_TOKEN_")) and (.value | startswith("${{github.actor}}"))) | .value | split(":")[1]')
+          OPTIC_TOKEN=$(echo '${{ steps.gettokens.outputs.tokens }}' | jq -R '. | fromjson | to_entries[] | select((.key | startswith("OPTIC_TOKEN_")) and (.value | startswith("${{github.actor}}"))) | .value | split(":")[1]')
+
+          echo ::add-mask::$NPM_TOKEN
+          echo ::add-mask::$OPTIC_TOKEN
+
+          echo USER_NPM_TOKEN=$(echo $NPM_TOKEN) >> $GITHUB_ENV
+          echo USER_OPTIC_TOKEN=$(echo $OPTIC_TOKEN) >> $GITHUB_ENV
```

Comment:
thanks for looking into ways to accomplish this. unfortunately, I don't think that asking users to include this code when using the action is a viable option, we need to come up with an alternative approach to support multiple tokens.